### DISCUSSION TITLE
Add OAuth Social Login for Telegram

### DISCUSSION THREAD

**Discussion by peiofour** [supabase/supabase#19363] [Feature Requests]
Telegram is a major instant messaging application. It has a login widget that allow to login to a website using Telegram OAuth.
This can be added to Supabase Social Login OAuth.

https://core.telegram.org/widgets/login

**Comment by ARHAEEM** [comment]
Many developers love Telegram and use it. However, once they start running a company, they shy away from Telegram integrations. This is because of the high user base, which could incur costs without financial benefits. Additionally, Western politics play a significant role; founders risk losing their funders when associated with an Arab or Russian-origin-based company.
